app/controllers/concerns/roles_controller.rb

```diff
@@ -25,6 +25,10 @@ def resource_class
     AccessControlList
   end
 
+  def policy_object
+    RoledControllerPolicy.new(api_user, resource_class, resource_name, action_name, params, scope_context: scope_context)
```

Comment:
feels like scope_context could just be a param assignment vs a method call here.

Reply:
memoization?

Reply:
The next branch I've already got nearly wrapped up actually extracts the additional options, and inlined the scope_context. Also adds memoization :)

Reply:
:tada: